keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,108 @@
+from keras import backend
+from keras.engine import base_layer
+from keras.engine import base_preprocessing_layer
+from keras.layers.preprocessing import preprocessing_utils as utils
+from keras.utils import image_utils
+from keras.utils import tf_utils
+import numpy as np
+import tensorflow as tf
+import random
+from tensorflow.python.util.tf_export import keras_export
+from tensorflow.tools.docs import doc_controls
+
+H_AXIS = -3
+W_AXIS = -2
+
+
+class RandomResizedCrop(tf.keras.internal.layers.BaseImageAugmentationLayer):
+    '''
+    This layer is used to
+    Crop a random portion of image and resize it to a given size.
+    A crop of the original image is made: the crop has a random area (H * W) and a random aspect ratio.
+    This crop is finally resized to the given size.
+    This is popularly used to train the Inception networks.
+    '''
+
+    def __init__(self, height, width, seed=None, **kwargs):
+        super().__init__
+        # Attribute to crop image
+        self.height = height
+        # Attribute to take the custom height
+        self.width = width
+        # Attribute to take the custom width
+        self.seed = seed
+        # Attribute to take random seed
+
+    #Methoid for taking the inputs
+    def call(self, inputs, training=True):
```

Comment:
You'll need to override `augment_image` instead of `call` when subclassing `BaseImageAugmentationLayer`

Reply:
sure sir working on it Asap!

Reply:
Done: Resolved it and applied ~~~augment_image~~~

Reply:
@LukeWood sir is the last commit all, correct? If not please let me know about it! Waiting to make the necessary changes. if required.